uat/features/stanford_date_formats.feature

```diff
@@ -1,10 +1,11 @@
+@stanford
 Feature: Stanford Date Formats
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford Date Formats module is working properly
 
 
-  @api
+  @api @safe @deploy
```

Comment:
And all of these I'm going to leave `@safe` in even before we fix them in the profile, because they're Mostly Harmless.